### ISSUE TITLE
pgo create user sometimes complains Error: the password format was invalid

### ISSUE THREAD

**Issue by tumluliu** [CrunchyData/postgres-operator#775]
**Describe the bug**
First of all, this doesn't happen every time, perhaps around 10% probability. But I think that's even worse and not easy to reproduce.

Here is the situation: when creating a new user with the command
```
pgo create user demo1 --selector name=democluster --db launchpad --managed --valid-days 180 --namespace pgo
``` 
It complains with
```
Error: the password format was invalid
```

My gut feeling is this message is from the underlying postgresql. And the reason could be the randomly generated password doesn't comply with some password format rule of pg. But unfortunately I can't find a clear password rule from pg's official documents. But anyway, this is just my guess. 

It would be great from your side to test running this role creation command in a loop with, e.g. 100 times. I think you will see this error a couple of times. My current workaround is just to let my ansible scripts keep trying until the creation succeeds, which is a pretty ugly solution.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Please tell us about your environment:**

* Operating System: macos / Azure k8s service
* Where is this running ( Local, Cloud Provider)
* Storage being used (NFS, Hostpath, Gluster, etc):
* Container Image Tag:
* PostgreSQL Version: 9.6.12
* Platform (Docker, Kubernetes, OpenShift): k8s
* Platform Version: 1.14.1
* PGO version: 4.0.0-rc10

**Additional context**

**Comment by cahoonpwork** [comment]
fix will be in 4.0.0-rc14.

**Comment by tumluliu** [comment]
Really looking forward to the 4.0.0 release! Many thanks!
